src/OpenTelemetry.AutoInstrumentation.Native/miniutfdata.h

```diff
@@ -19,6 +19,8 @@
  * THE SOFTWARE.
  */
 
+#include <stdint.h>
```

Comment:
It seems that the docker upgrades changed the standard includes to a more recent version...